src/renderer/i18n/en/deposit.ts

```diff
@@ -35,6 +35,13 @@ const deposit: DepositMessages = {
   'deposit.poolDetails.allTimeVal': 'All time volume',
   'deposit.poolDetails.totalSwaps': 'Total swaps',
   'deposit.poolDetails.totalUsers': 'Total users',
+  'deposit.poolDetails.liquidity': 'Liquidity',
+  'deposit.poolDetails.volumn': 'Volumn',
+  'deposit.poolDetails.earnings': 'Earnings',
+  'deposit.poolDetails.fees': 'Fees',
+  'deposit.poolDetails.totalTx': 'Total Tx',
+  'deposit.poolDetails.members': 'Members',
+  'deposit.poolDetails.apy': 'APY',
```

Comment:
It seems we have some some of translations at other places as well (e.g. `Liquidity`, `Earnings`, `Fees` etc.). If so, pls extract duplications to `common.xyz`